apps/aecore/lib/aecore/chain/block_validation.ex

```diff
@@ -113,6 +117,11 @@ defmodule Aecore.Chain.BlockValidation do
     Enum.map(txs, fn tx -> :ok == SignedTx.validate(tx) end)
   end
 
+  @spec check_is_minimum_fee_met?(list()) :: boolean()
+  def check_is_minimum_fee_met?(txs) do
+    Enum.all?(txs, fn tx -> Pool.is_minimum_fee_met?(tx, :pool) end)
```

Comment:
Why isn't identifier (second argument to `is_minimum_fee_met?`) `:validation`?

Reply:
I will change to `:validation`